src/medCore/database/medDatabaseModel.cpp

```diff
@@ -143,7 +137,7 @@ medDatabaseModel::medDatabaseModel(QObject *parent, bool justBringStudies) : QAb
     d->seAttributes[i++] = medMetaDataKeys::Performer.key();
     d->seAttributes[i++] = medMetaDataKeys::Institution.key();
     d->seAttributes[i++] = medMetaDataKeys::Report.key();
-
+    d->seAttributes[i++] = medMetaDataKeys::ThumbnailPath.key();
```

Comment:
To explain a bit, this line adds the thumbnail location of a volume to the attributes of a data. A thumbnail is an exported image in .png (or jpg, not sure right now). When you move a data to an other patient, you need to update the real location of the thumbnail.